tracer/build/_build/Build.GitHub.cs

```diff
@@ -81,6 +81,40 @@ partial class Build
             Console.WriteLine($"PR assigned");
         });
 
+    Target CloseMilestone => _ => _
+       .Unlisted()
+       .Requires(() => GitHubToken)
+       .Requires(() => Version)
+       .Executes(async() =>
+       {
+            var client = GetGitHubClient();
+
+            var milestone = await GetMilestone(client, Version);
+            if (milestone is null)
+            {
+                Console.WriteLine($"Milestone {Version} not found. Doing nothing");
+                return;
+            }
+
+            Console.WriteLine($"Closing {milestone.Title}");
+
+            try
+            {
+                await client.Issue.Milestone.Update(
+                    owner: GitHubRepositoryOwner,
+                    name: GitHubRepositoryName,
+                    number: milestone.Number,
+                    new MilestoneUpdate { State = ItemState.Closed });
+            }
+            catch (ApiValidationException)
+            {
+                Console.WriteLine($"Unable to close {milestone.Title}.");
```

Comment:
Maybe write the exception message?

Reply:
Added in a0840de92f25173116e71b837a45e91dc6ad66a4